dist/linux/appimage/build.sh

```diff
@@ -11,31 +11,36 @@ command -v curl >/dev/null 2>&1 || { echo >&2 "curl not found."; exit 1; }
 VERSION=$(mvn -f ../../../pom.xml help:evaluate -Dexpression=project.version -q -DforceStdout)
 SEMVER_STR=${VERSION}
 
+mvn -f ../../../pom.xml versions:set -DnewVersion=${SEMVER_STR}
+
 # compile
-mvn -B -f ../../../pom.xml clean package -DskipTests -Plinux
+mvn -B -f ../../../pom.xml clean package -Pdependency-check,linux -DskipTests
```

Comment:
Running the dependency check is really just a CI job to avoid publishing changes with vulnerable dependencies. It doesn't add much benefit for local builds, except if you want to fail if a CVE was added in the meantime.